EIPS/eip-1313.md

```diff
@@ -0,0 +1,62 @@
+---
+eip: 1313
```

Comment:
```suggestion
eip: 5344
```
Also rename the file.

Reply:
Thank you for lot of feed back, I will update it.